File: 프로그래머스/1/133499. 옹알이 （2）/옹알이 （2）.java
class Solution {
    String[] arr = {"aya", "ye", "woo", "ma"};
    public int solution(String[] babbling) {
        int answer = 0;
        for(String s : babbling){
            String temp = s;
            for(String ss : arr){
                temp = temp.replaceAll("(?<!"+ ss +")"+"("+ ss + ")" +"(?!" + ss + ")", " ");
            }
            temp = temp.replaceAll(" ", "");
            System.out.println(temp);
            if(temp.equals("")) answer++;
        }
        
        return answer;
    }
}
/*
같은 발음이 연속되지 않고 

*/

File: 프로그래머스/1/133499. 옹알이 （2）/옹알이 （2）.py
def solution(babbling):
    temp = ["aya", "ye", "woo", "ma"]
    ret = 0
    for bab in babbling:
        flag = True
        for word in temp:
            if word*2 in  bab:
                flag = False
                break
        
        if flag:
            for word in temp:
                bab = bab.replace(word, ' ')
            
            bab = bab.replace(' ', '')
            if not bab:
                ret += 1
    return ret
        
        
                
                
                
"""
연속을 어떻게 체크하지??


"""